mesonbuild/dependencies/misc.py

```diff
@@ -574,3 +576,50 @@ def get_methods(self):
             return [DependencyMethods.PKGCONFIG, DependencyMethods.EXTRAFRAMEWORK]
         else:
             return [DependencyMethods.PKGCONFIG]
+
+class PcapDependency(ExternalDependency):
+    def __init__(self, environment, kwargs):
+        super().__init__('pcap', environment, None, kwargs)
+        if DependencyMethods.PKGCONFIG in self.methods:
+            try:
+                kwargs['required'] = False
+                pcdep = PkgConfigDependency('pcap', environment, kwargs)
+                if pcdep.found():
+                    self.type_name = 'pkgconfig'
+                    self.is_found = True
+                    self.compile_args = pcdep.get_compile_args()
+                    self.link_args = pcdep.get_link_args()
+                    self.version = pcdep.get_version()
+                    return
+            except Exception as e:
+                mlog.debug('Pcap not found via pkgconfig. Trying next, error was:', str(e))
+        if DependencyMethods.PCAPCONFIG in self.methods:
+            pcapconf = shutil.which('pcap-config')
+            if pcapconf:
+                stdo = Popen_safe(['pcap-config', '--cflags'])[1]
+                self.compile_args = stdo.strip().split()
+                stdo = Popen_safe(['pcap-config', '--libs'])[1]
+                self.link_args = stdo.strip().split()
+                self.version = '0'
```

Comment:
Well, since this got merged I'll look into fixing this after the release. The version being wrong is low-impact since version checks are broken anyway.